util/checker/src/cmd/check_stubs_compile.rs

```diff
@@ -0,0 +1,169 @@
+use std::{fs, path::PathBuf, process::Command};
+
+fn make_reserve_copies(modified_exercise: &PathBuf) {
+    let stub_path = modified_exercise.join("src").join("lib.rs");
+
+    let stub_path_copy = modified_exercise.join("src").join("lib.rs.orig");
```

Comment:
Why specify `*_copy` variables manually instead of just using [`with_extension`](https://doc.rust-lang.org/std/path/struct.Path.html#method.with_extension)?